pageserve/app.py

```diff
@@ -164,13 +164,21 @@ def add_event():
         Redirect to index if 201 response from events service.
         Error message and status 400 otherwise.
     """
-    url = app.config['EVENTS_ENDPOINT'] + 'add'
-    form_data = dict(**request.form.to_dict(), author_id=get_user()['user_id'])
-    r = requests.post(url, data=form_data)
-    if r.status_code == 201:
-        # upload successful, redirect to index
-        return redirect(url_for("index"))
-    return r.content, r.status_code
+    try:
+        user = get_user()
+        if not user['is_organizer']:
```

Comment:
I think this throws an unhandled TypeError if the user isn't logged in here.

Reply:
That throws a `KeyError` if the user isn't logged in (`user` is `None`) which is caught on line 180.

Reply:
Done https://github.com/knative-portability/large-events/pull/62/commits/d2d4d3477d13ac7079f75e694564a02f23be8c03